packages/ui/src/components/Switch.tsx

```diff
@@ -75,9 +75,9 @@ export default styled(Switch)(({ theme }: ThemeProps) => `
       width: 16px;
```

Comment:
There are a couple of minor adjustments that can be done here, that will end up perfectly matching the switch to the designs:

- `width` and `height` should be `18px`
- `left` should match `bottom` for `3px`
- add `box-shadow: 0px 1px 3px rgba(21, 41, 53, 0.12), 0px 1px 2px rgba(21, 41, 53, 0.24);` — had to fish that out of from the depths of the Figma designs

Otherwise, this switch is already pretty much there!

Reply:
> box-shadow: 0px 1px 3px rgba(21, 41, 53, 0.12), 0px 1px 2px rgba(21, 41, 53, 0.24);

That alone makes a world of difference 😱

Reply:
✨